app_dart/lib/src/request_handlers/flaky_handler_utils.dart

```diff
@@ -77,7 +79,7 @@ Flaky builds:
 ${_issueBuildLinks(builder: statistic.name, builds: statistic.flakyBuilds!)}
 
 Succeeded builds (3 most recent):
-${_issueBuildLinks(builder: statistic.name, builds: statistic.succeededBuilds!.sublist(0, 3))}
+${_issueBuildLinks(builder: statistic.name, builds: statistic.succeededBuilds!.sublist(0, statistic.succeededBuilds!.length >= kSuccessBuildNumberLimit ? kSuccessBuildNumberLimit : statistic.succeededBuilds!.length))}
```

Comment:
This is hard to follow. Can we assign this to a variable and pass it instead?

Reply:
Updated.